buildpack.toml

```diff
@@ -10,7 +10,4 @@ api = "0.4"
   pre-package = "./scripts/build.sh"
 
 [[stacks]]
-  id = "io.buildpacks.stacks.bionic"
-
-[[stacks]]
-  id = "org.cloudfoundry.stacks.cflinuxfs3"
+  id = "*"
```

Comment:
@ybykov-a9s This apparently means this buildpack is compatible to all stacks. I wonder if that is such a good idea. Aren't there stacks with which this buildpack won't work?

Reply:
@khaledavarteq Paketo team moves this direction. I just try to maintain the codebase as close to the original one as possible. My opinion that they're just preparing to changing base stack to 20.04.  cflinuxfs3 stack should work with this buildpack. Nothing was changed to break the compatibility.